.github/ISSUE_TEMPLATE/bug.yml

```
@@ -25,6 +25,11 @@ body:
           required: true
         - label: I have resolved all warnings from `brew doctor` and that did not fix my problem.
           required: true
+        - label: I have unset `HOMEBREW_{BOTTLE,ARTIFACT}_DOMAIN` and `HOMEBREW_{BREW,CORE}_GIT_REMOTE` and am still able to reproduce this issue.
```

Might be best to separate this because I don't know if all users know what `{foo,bar}` means. Probably don't need four checkboxes, but might be good to change to:

```suggestion
        - label: I have not set `HOMEBREW_BOTTLE_DOMAIN`, `HOMEBREW_ARTIFACT_DOMAIN`, `HOMEBREW_BREW_GIT_REMOTE`, or `HOMEBREW_CORE_GIT_REMOTE`  and am still able to reproduce this issue.
```

I also changed "unset" to "not set" because I don't want users who have never touched these to be confused about how to unset something they haven't set (and might not know where to set/unset in the first place).

Just suggestions and happy to accept other variations for sure